tests/core/test_module.py

```diff
@@ -474,13 +402,13 @@ class DummyLocal(caikit.core.blocks.base.BlockBase):
         pass
 
     @caikit.core.blocks.block(backend_type=backend_types.BAZ, base_module=DummyLocal)
-    class DummyBaz(caikit.core.ModuleBase):
+    class DummyBaz(caikit.core.blocks.base.BlockBase):
         pass
 
     @caikit.core.blocks.block(
         backend_type=backend_types.FOO, base_module=DummyLocal.MODULE_ID
     )
-    class DummyFoo(caikit.core.ModuleBase):
+    class DummyFoo(caikit.core.blocks.base.BlockBase):
         pass
 
     assert DummyLocal in list(caikit.core.MODULE_REGISTRY.values())
```

Comment:
I think we need to add a new test to exercise the ability to disambiguate multiple instances of the same backend by name. I'll add that now